src/content/en/2019/accessibility.md

```diff
@@ -73,105 +72,108 @@ In order to facilitate this feature, the translation engines need to know what l
 
-You can add captions to all video and audio by using the [`<track>`](https://developer.mozilla.org/en-US/docs/Web/Guide/Audio_and_video_delivery/Adding_captions_and_subtitles_to_HTML5_video) tag.
+You can add captions to all video and audio content by using the [`<track>`](https://developer.mozilla.org/en-US/docs/Web/Guide/Audio_and_video_delivery/Adding_captions_and_subtitles_to_HTML5_video) tag.
 
 ## Ease of page navigation
 
-When you open the menu in a restaurant, the first thing you probably do is read all of the section headers: appetizers, salads, main course, and dessert. This allows you to scan a menu for all of the options and jump quickly to the dishes that are most interesting to you. Similarly, when a visitor opens a web page, their goal is to find the information they are most interested in: the reason they came to the page in the first place. In order to help users find their desired content as fast as possible (and prevent them from hitting the back button), we try to separate the contents of our pages into several visually distinct sections. For example: a site header for navigation, various headings in our articles so users can quickly scan them, a footer for other extraneous resources, and more.
+When you open the menu in a restaurant, the first thing you probably do is read all of the section headers: appetizers, salads, main course, and dessert. This allows you to scan a menu for all of the options and jump quickly to the dishes most interesting to you. Similarly, when a visitor opens a web page their goal is to find the information they are most interested in: the reason they came to the page in the first place. In order to help users find their desired content as fast as possible (and prevent them from hitting the back button), we try to separate the contents of our pages into several visually distinct sections. For example: a site header for navigation, various headings in our articles so users can quickly scan them, a footer for other extraneous resources, and more.
 
 While this is exceptionally important, we need to take care to mark up our pages so our visitors’ computers can perceive these distinct sections as well. Why? While most of us use a mouse to navigate pages, many others rely on keyboards and screen readers. These technologies rely heavily on how well their computers understand your page.
 
-So how well are we doing? Very poorly. **Only 26.03% of sites** even contained the most basic part of navigation: a [main landmark](#heading=h.23wv35354sk).
+So how well are we doing? Very poorly. Only 26.03% of sites even contained the most basic part of navigation: a main landmark.
```

Comment:
For a more accurate idea I could edit the 09.02 metric. And measure how many sites markup their header, footer, navigation and footer.

This would be a more representative metric.

Reply:
It's late so let's keep the metrics as-is and try to rephrase if possible. Would you be comfortable removing this sentence? It's still discussed below but it won't be the sole metric judging the a11y space.